Source/WebApi.HypermediaExtensions/WebApi/Formatter/SirenConverter.cs

```diff
@@ -166,17 +166,14 @@ private void AddActionFields(JObject jAction, Type actionParameterType)
                 {"type", DefaultMediaTypes.ApplicationJson}
             };
 
-            string classRoute;
-            if (!routeResolver.TryGetRouteByType(actionParameterType, out classRoute))
+            if (!routeResolver.TryGetRouteByType(actionParameterType, out var classRoute))
             {
-                classRoute = actionParameterType.BeautifulName();
+                classRoute = routeResolver.RouteUrl(RouteNames.ActionParameterTypes, new { parameterTypeName = actionParameterType.BeautifulName() });
```

Comment:
What if the route can not be resolved, e.g. Auto type routes disabled? It was using the class name as fallback.

Reply:
hm... the ActionParameterTypes controller would be there anyway, but it would return 404 for every parameter type. Is that ok?